### PR TITLE
Add analytics events on jobs page

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 1]
```diff
@@ -177,22 +181,36 @@ export default {
...
+    window.removeEventListener("scroll", this.sendEvent);
+  },
   computed: {
     ...mapState(useJobListStore, {
       careers: "items",
       jobsError: "error",
     }),
   },
   methods: {
+    sendEvent() {
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:199]: Do we need this here?

File: vue-frontend/src/components/jobs/VirtuesView.vue [hunk 2]
```diff
@@ -208,70 +215,95 @@ export default {
           image: virtue1,
           description:
             "Sometimes we fail, but failure makes us smarter. We get back up, we make the next bet, and we go!",
+          event: "hover_virtue",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/VirtuesView.vue:218]: As all events name are same, we can directly apply

File: vue-frontend/src/components/jobs/WhyCanopas.vue [hunk 3]
```diff
@@ -91,13 +93,20 @@ export default {
     };
   },
   mounted() {
-    window.addEventListener("scroll", this.handleScroll);
+    window.addEventListener("scroll", this.handleScroll, this.sendEvent);
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/WhyCanopas.vue:96]: Is this working?
Also we don't need it as you have already added refs and same code in index page

File: vue-frontend/src/pages/jobs/[id].vue [hunk 4]
```diff
@@ -147,10 +150,20 @@ export default {
   },
   mounted() {
     this.setCareerDetails();
+    window.addEventListener("scroll", this.sendEvent);
     this.$gtag.event("view_page_job_detail");
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/jobs/[id].vue:154]: Remove `view_page_job_detail` event

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/utils.js [hunk 5]
```diff
@@ -17,6 +17,15 @@ const events = {
     1: "view_home_togness_portfolio",
     2: "view_home_justly_portfolio",
     3: "view_home_smile_portfolio",
+    landingview: "view_jobs_landing_section",
+    virtue: "view_virtues_section",
+    perks: "view_perks_benefits",
+    whycanopas: "view_why_canopas",
+    joblist: "view_jobs_list",
+    readmore: "view_job_read_more",
```
[the remark this answers is not in the document]
Comment (cp-priyanka-g) [reply vue-frontend/src/utils.js:25]: Please check on  pages/jobs/[id].vue  line 75

File: vue-frontend/src/components/jobs/VirtuesView.vue [hunk 6]
```diff
@@ -197,9 +201,11 @@ import virtue7 from "@/assets/images/virtues/jobs_canopas_virtue_7.svg";
 import virtue8 from "@/assets/images/virtues/jobs_canopas_virtue_8.svg";
 import virtue9 from "@/assets/images/virtues/jobs_canopas_virtue_9.svg";
 import virtue10 from "@/assets/images/virtues/jobs_canopas_virtue_10.svg";
+
 export default {
   data() {
     return {
+      event: "",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/VirtuesView.vue:208]: Remove this

File: vue-frontend/src/components/jobs/WhyCanopas.vue [hunk 7]
```diff
@@ -90,13 +90,6 @@ export default {
       ],
     };
   },
-  mounted() {
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/WhyCanopas.vue:93]: I think we need `handleScroll`

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 8]
```diff
@@ -100,38 +100,45 @@ export default {
           className:
             " tw-mt-0 tw-mb-0 tw-ml-0 tw-p-0 tw-text-black-900/80 hover:tw-text-black-900/80 tw-mr-[20px] sm:tw-mr-[40px] tw-inline-block tw-relative after:tw-absolute after:tw-w-2/4 after:tw-scale-x-0 after:tw-h-0.5 after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-black-900 after:tw-origin-bottom-left after:tw-duration-300 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left",
           target: false,
+          event: this.isJobsUrl ? "tap_jobs_header_home" : "",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:103]: Will it send event if it is  " "?

File: vue-frontend/src/components/jobs/FaqSection.vue [hunk 9]
```diff
@@ -200,18 +202,15 @@ export default {
     CollapseTransition,
     FontAwesomeIcon,
   },
-  mounted() {
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/FaqSection.vue:203]: I think we need `handleScroll`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/jobs/PerksAndBenifits.vue [hunk 10]
```diff
@@ -84,6 +88,7 @@ import celebration from "@/assets/images/benifits/jobs_canopas_celebration.svg";
 export default {
   data() {
     return {
+      event: "",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/PerksAndBenifits.vue:91]: Remove this.
Once any comment is added, then check for all occurrences. It's not about just that code only.

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 11]
```diff
@@ -100,38 +100,45 @@ export default {
...
+          event: "tap_jobs_header_home",
         },
         {
           name: "Career",
           url: "/jobs",
           className:
             "tw-mt-0  tw-mb-0 tw-ml-0 tw-p-0 tw-text-black-900/80 hover:tw-text-black-900/80 tw-mr-[20px] sm:tw-mr-[40px] tw-inline-block tw-relative after:tw-absolute after:tw-w-2/4 after:tw-scale-x-0 after:tw-h-0.5 after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-black-900 after:tw-origin-bottom-left after:tw-duration-300 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left",
           target: false,
-          event: "tap_header_view_career_page",
+          event: "tap_header_view_career_page ",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:111]: Remove space

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 12]
```diff
@@ -100,38 +100,45 @@ export default {
           className:
             " tw-mt-0 tw-mb-0 tw-ml-0 tw-p-0 tw-text-black-900/80 hover:tw-text-black-900/80 tw-mr-[20px] sm:tw-mr-[40px] tw-inline-block tw-relative after:tw-absolute after:tw-w-2/4 after:tw-scale-x-0 after:tw-h-0.5 after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-black-900 after:tw-origin-bottom-left after:tw-duration-300 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left",
           target: false,
+          event: "tap_jobs_header_home",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:103]: You have to check `isJobsUrl` here.
But have to add condition in `sendEvent` that `event != " "`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-shruti-s — APPROVED** [review]

File: vue-frontend/src/utils.js [hunk 13]
```diff
@@ -17,6 +17,15 @@ const events = {
     1: "view_home_togness_portfolio",
     2: "view_home_justly_portfolio",
     3: "view_home_smile_portfolio",
+    landingview: "view_jobs_landing_section",
```
Comment (cp-sumi-k) [inline vue-frontend/src/utils.js:20]: Should it be inside the `portfolios`?

File: vue-frontend/src/pages/jobs/index.vue [hunk 14]
```diff
@@ -73,6 +76,27 @@ export default {
     FaqSection,
     ScreenFooter2,
   },
+  data() {
+    return {
+      event: "",
+    };
+  },
+  mounted() {
+    this.$gtag.event("view_page_jobs");
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/jobs/index.vue:85]: Remove this event,as we have added event component wise

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**